templates/artist/includes/artist_detail_overview.html

```diff
@@ -4,140 +4,125 @@
+                {% endif %}
+
+            <!-- What do you get for investing? -->
+            <h3>WHAT DO YOU GET FOR INVESTING?</h3>
+            <div id="canvas-holder">
+                <canvas id="chart-area" width="500" height="500"></canvas>
+            </div>
             <ul>
-                {% for expense in campaign.expense_set.all %}
-                    <li>{{ expense.expense }}</li>
-                {% endfor %}
-                <li><strong>Total = ${{ campaign.amount|intcomma }}</strong></li>
-            </ul>
-        {% endif %}
-        <div class="break">
-            <hr>
-        </div>
-        <!-- What do you get for investing? -->
-        <h3>WHAT DO YOU GET FOR INVESTING?</h3>
-        <p>We are raising ${{ campaign.amount|intcomma }} {{ campaign.project.reason }} and giving {{ campaign.fans_percentage }}% of the sales back to the people who invest in it.</p>
-        <div class="break">
-            <hr>
-        </div>
-        <!-- Percentage -->
-        <h2>PERCENTAGE</h2>
-        <div id="canvas-holder">
-            <canvas id="chart-area" width="500" height="500"></canvas>
-        </div>
-        <ul>
-            {% for percentage_breakdown in campaign.project.artist_percentage %}
-                <li>{{ percentage_breakdown.name }}: {{ percentage_breakdown.percentage|notrail_floatformat:2 }}%</li>
-            {% endfor %}
-            <li>
-                {% if user_investor %}Other Investors:{% else %}Investors:{% endif %}
-                {{ fans_percentage|notrail_floatformat:2 }}%
-            </li>
-            {% if user_investor %}
-                <li>
+                {% if user_investor %}
+                <li class="me-investing">
                     Me:
                     {% if user_investor.percentage < 0.005 %}
                         <0.01%
                     {% else %}
                         {{ user_investor.percentage|notrail_floatformat:2 }}%
                     {% endif %}
                 </li>
+                {% for percentage_breakdown in campaign.project.artist_percentage %}
+                    <li>{{ percentage_breakdown.name }}: {{ percentage_breakdown.percentage|notrail_floatformat:2 }}%</li>
+                {% endfor %}
+                <li>
+                    {% if user_investor %}Other Investors:{% else %}Investors:{% endif %}
+                    {{ fans_percentage|notrail_floatformat:2 }}%
+                </li>
+                {% endif %}
+            </ul>
+            <h4 class="investing">You'll also get</h4>
+            <p class="investing-copy">To hear the music before anyone else</p>
+            <p class="investing-copy">To download the project as soon as it's ready</p>
+            <p class="investing-copy">Exclusive updates from the artist</p>
+            <div class="break">
+                <hr>
+            </div>
+            <!-- What is the money being used for? -->
+            {% if campaign %}
```

Comment:
We can merge this if statement, and the one below with `{% if campaign and campaign.project.generated_revenue %}`. Once we change that, we should remove the inner `{% endif %}`.